modules/components/pages/inputs/amqp_1.adoc

```diff
@@ -199,12 +199,8 @@ Requires version 3.45.0 or newer
 === `tls.root_cas`
 
 An optional root certificate authority to use. This is a string, representing a certificate chain from the parent trusted root certificate, to possible intermediate signing certificates, to the host certificate.
-[CAUTION]
-====
-This field contains sensitive information that usually shouldn't be added to a config directly, read our xref:configuration:secrets.adoc[secrets page for more info].
-====
-
 
+include::components:partial$secret_warning_sm.adoc[]
```

Comment:
Use the single partial that I mentioned here: https://github.com/redpanda-data/rp-connect-docs/pull/29/files#r1745115665

then even if this gets included in cloud one day, the content is ready.

Reply:
this goes for any other standalone includes of the SM secret partial